src/blocktypes/microvms.nix

```diff
@@ -17,17 +15,18 @@
       flake,
       fragment,
       fragmentRelPath,
+      target,
     }: let
-      run = ["nix" "run" "${flake}#${fragment}.config.microvm.runner"];
+      run = import ./actions/run.nix {
+        inherit target lib;
+      };
     in [
       {
         name = "microvm";
         description = "exec this microvm";
-        command =
-          (l.concatStringsSep "\t" run)
-          + ".$(nix eval --json --option warn-dirty false\ "
-          + "${flake}#${fragment}.config.microvm.hypervisor)"
-          + "\ ${substituters} ${keys}";
+        command = ''
+          ${run target.config.microvm.runner.${target.config.microvm.hypervisor}}
```

Comment:
@GTrunSec may you have a chance to confirm this still works? Caveat: substituters above where squeezed out of the action interface.

Reply:
I tested a basic example and it built fine, so I think we should be clear, but any confirmation is great.

Reply:
I got an error output with `cannot coerce a set to a string`. Also, Do not remove substitutes options, the microvm is built-in a patched Linux kernel.

Reply:
They should be in your flake.nix instead, that is the CI contract and you wouldn't want it not to consider than anyway.